doc/api/util.md

```diff
@@ -181,25 +181,25 @@ The `--throw-deprecation` command line flag and `process.throwDeprecation`
 property take precedence over `--trace-deprecation` and
 `process.traceDeprecation`.
 
-## util.format(format[, ...args])
+## util.format([format][, ...args])
 <!-- YAML
 added: v0.5.3
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: The output string's formatting will again dependent on the
+                 type of the first argument. This reverts
+                 https://github.com/nodejs/node/pull/23162.
+  - version: v12.0.0
+    pr-url: https://github.com/nodejs/node/pull/23162
+    description: The output string's formatting is no longer dependent on the
+                 type of the first argument. This change removes previously
+                 present quotes from strings that were being output when the
+                 first argument was not a string.
   - version: v11.4.0
     pr-url: https://github.com/nodejs/node/pull/23708
     description: The `%d`, `%f` and `%i` specifiers now support Symbols
                  properly.
-  - version: v12.0.0
-    pr-url: https://github.com/nodejs/node/pull/23162
-    description: The `format` argument is now only taken as such if it actually
-                 contains format specifiers.
-  - version: v12.0.0
-    pr-url: https://github.com/nodejs/node/pull/23162
-    description: If the `format` argument is not a format string, the output
-                 string's formatting is no longer dependent on the type of the
-                 first argument. This change removes previously present quotes
-                 from strings that were being output when the first argument
-                 was not a string.
```

Comment:
Can you keep these?

Reply:
I removed the `format` description, since it does not apply as I read the spec. The description is therefore confusing to me. I am fine to keep it though, if you think that's best.

Reply:
I feel like the changelog should keep some indication that there was a change and what it affected, yes

Reply:
Did you see that there where two changes entries for the same PR and that I kept one of them (a bit shorter)?

Reply:
Ah, I didn’t see that you also moved these to be in the right order. In that case I’m good, thanks! :+1: